src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -63,14 +63,16 @@ requires the CHERI RISC-V hart to support {cheri_base_ext_name} and
 
 The _<<cheri_execution_mode>>_ is determined by a bit in the metadata of the <<pcc>> called the <<m_bit>>.
 {cheri_default_ext_name} adds a new _<<cheri_execution_mode>>_ field (M) to the capability format.
-Although always present, it only takes effect in code capabilities, i.e., when the <<x_perm>> is set.
+This field _need_ be present only in capabilities granting <<X-permission>>,
+as it is only ever architecturally interpreted on the capability resident in <<pcc>>.
+Capabilities not granting <<X-permission>> may or may not have a defined M field,
+and attempting to update this field may or may not clear their {ctag}s.
 The exact location of the M-bit in the capability format for XLEN=32 and XLEN=64 is described in <<app_cap_description>>.
 
 * Mode (M)={CAP_MODE_VALUE} indicates {cheri_cap_mode_name}.
 * Mode (M)={INT_MODE_VALUE} indicates {cheri_int_mode_name}.
```

Comment:
Should I push this information into the format chapter in #845 ?

Reply:
I read these as being the "abstract" values of the M field as returned by the `YMODER` getter and sent to the `YMODEW` setter, so I think these lines can stay here?

Reply:
Yeah it happens to be both. I'll add a note saying something like "the architectural view of the mode" just to be clear when I get back to updating #845 .

Reply:
yes - good point - the capability encoding doesn't _have to_ encode the M-bit that way